### ISSUE TITLE
Error running XLNet if there are variable number of options and not 4 options always

### ISSUE THREAD

**Issue by Sumegh-git** [zihangdai/xlnet#61]
This is the error : 
Key: input_mask.  Can't parse serialized Example.
	 [[{{node ParseSingleExample/ParseSingleExample}}]]
	 [[input_pipeline_task0/while/IteratorGetNext]]

Can you suggest what changes to make? 
I am using a custom dataset similar to `RACE` also formatted using `preprocess.py` from the repo `RACE_AR_BASELINES`.
Since my dataset has variable number of options. Some questions have 2, some have 4,etc..

**Comment by kimiyoung** [comment]
`preprocess.py` is for pretraining. For finetuning on RACE, please refer to `run_race.py`. I think the easiest way might be to add some pseudo padding candidates to the input so that all questions have the same number of candidate options.
